scripts/tools/nrfconnect/generate_nrfconnect_chip_factory_data.py

```diff
@@ -102,7 +101,7 @@ def generate_json(self):
         else:
             rd_uid = self._args.rd_uid
         if not self._args.spake2_verifier:
-            spake_2_verifier = base64.b64decode(self._generate_spake2_verifier()).hex()
+            spake_2_verifier = self._generate_spake2_verifier()
```

Comment:
So now it will be put to factory data as a string? The tool generates BASE64 as far as I remember.

Reply:
Yes, it was my fault because in Matter it was given as an ASCII string by default, and I realized that in this way commissioning doesn't work. Here is the default example: https://github.com/project-chip/connectedhomeip/blob/master/src/include/platform/CHIPDeviceConfig.h#L974

Reply:
`GetSpake2pSalt` returns a ByteSpan, so we can put raw byte string in the factory data. The legacy commissionable provider decodes B64, but ours doesn't have to do it, so we can use more compact version.